providers/closecrm/types.go

```diff
@@ -0,0 +1,40 @@
+package closecrm
+
+import (
+	"time"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/handy"
+)
+
+type SearchParams struct {
+	ObjectName string
+	Fields     handy.StringSet
```

Comment:
This can be just []string. Unnecessary use of this set. There is no logic in this PR that uses the benefits of this package. I actually don't like the abuse of this handy package happening in this repo. Please refrain from using it unless it is really necessary.